lib/common/vm-common.js

```diff
@@ -12,11 +12,11 @@
  * A brief overview of this source file: what is its purpose.
  */
 
-
-var assert = require('assert');
+var assert = require('assert-plus');
 var restify = require('restify');
 
 var clone = require('./util').clone;
+var validation = require('./validation');
```

Comment:
Do we actually use `"validation"` in this file?  Actually all of
the changes in this file seem pretty superficial -- I would
probably just leave the file alone, or do them in another
bug/change, to make the change a bit cleaner.